src/app/lib/config/services/afaanoromoo.js

```diff
@@ -140,15 +140,15 @@ export const service = {
       lastUpdated: 'Yeroo dhuma kan haaromfame: ',
     },
     footer: {
+      trustProjectLink: {
+        href: 'https://www.bbc.com/afaanoromoo/institutional-49281861',
+        text: 'BBC News maaliif amanuu dandeessa',
+      },
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
         text: 'Gara geessituu alaatti akkaataa itti hojjennu dubbisi.',
       },
       links: [
-        {
-          href: 'https://www.bbc.com/afaanoromoo/institutional-49281861',
-          text: 'BBC News maaliif amanuu dandeessa',
-        },
```

Comment:
We need to keep the trust project link in the generic link array for now otherwise it will disappear from all services. Once we have implemented the prop to pull this new config in, we can remove it from links.